README.md

```diff
@@ -1,54 +1,57 @@
 # Pokémon Sprite and Image Dataset for Conditional Generation
 
-This dataset contains sprites, animated sprites, Bulbapedia artworks, and tabular data for all of the Pokémon listed on Bulbapdia. I originally built this dataset to [train a diffusion model](https://github.com/KyroChi/pokemon_sprite_generator) to generate new Pokémon sprites.
+[](https://github.com/ShanNicChi/pokemon-sprite-dataset#pok%C3%A9mon-sprite-and-image-dataset-for-conditional-generation)
 
-This dataset is built on the back of the [Bulbapeida](https://bulbapedia.bulbagarden.net/wiki/Main_Page) and the [PokéAPI](https://pokeapi.co/). It aggregates the resources that I required for building a generative diffusion model for Pokémon sprite animations. 
+This dataset contains sprites, animated sprites, Bulbapedia artworks, and tabular data for all of the Pokémon listed on Bulbapdia. We built this dataset to  [train a diffusion model](https://github.com/KyroChi/pokemon_sprite_generator)  to generate new Pokémon sprites.
```

Comment:
Maybe combine this line with the next line so we don't repeat about the diffusion model?